### DISCUSSION TITLE
App ends up with embedded frameworks from Firebase

### DISCUSSION THREAD

**Discussion by macdrevx** [firebase/firebase-ios-sdk#13951] [Q&A]
I noticed recently that when firebase is installed via SPM, the final app bundle ends up containing some embedded fameworks:

- FirebaseAnalytics.framework
- GoogleAppMeasurement.framework
- GoogleAppMeasurementIdentitySupport.framework

Why do these exist? Don't the corresponding binary targets refer to static frameworks? (I'm on 10.23.1)

**Comment by paulb777** [comment]
This question might be better directed towards Apple since its a change in their tooling. All static frameworks are being wrapped into dynamic ones in the app build tooling.

There is a somewhat related open question being discussed at https://forums.developer.apple.com/forums/thread/758816
